recipes/denoptim/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_targets:
```

Comment:
No longer needed because latest release is stable.

Reply:
Label removed in 3d20752eca6409b6cd5cbd51009a597700470e11